devtools/cli/distribution/release-cli.sh

```diff
@@ -35,9 +42,10 @@ tar -zcf ${JAVA_BINARY_DIR}.tar.gz ${JAVA_BINARY_DIR}
 popd
 
 export JRELEASER_PROJECT_VERSION=${VERSION}
-export JRELEASER_BRANCH=main
+export JRELEASER_BRANCH=${BRANCH}
+export JRELEASER_CHOCOLATEY_GITHUB_BRANCH=${BRANCH}
 
-jbang jreleaser@jreleaser full-release \
+jbang jreleaser.java full-release \
```

Comment:
Can you specify a tag here instead of having the main class? @maxandersen

Reply:
Maybe.

Doc talks about branches here: https://www.jbang.dev/documentation/guide/latest/alias_catalogs.html . Maybe it also works for tags, I don't know.

Reply:
How about using the GAV here instead? The jreleaser.java in the jbang-catalog doesn't seem to do much TBH